src/sipp.cpp

```diff
@@ -1134,6 +1134,7 @@ void sipp_exit(int rc, int rtp_errors, int echo_errors)
         // then everything is OK!
         if ((rtp_errors > 0) || (echo_errors > 0))
         {
+            ERROR("GOT rtp_errors = %d and echo_errors = %d", rtp_errors, echo_errors);
```

Comment:
ERROR() does not return, so EXIT_RTPCHECK_FAILED is not returned to the OS. You'd want WARNING.

Maybe `"Got %d rtp_errors and %d echo_errors"` for more pleasant english.